lib/openstack/openstack_handle/handle.rb

```diff
@@ -25,16 +25,19 @@ class Handle
       "Orchestration" => :orchestration
     }
 
+    def self.try_connection_with_ssl
```

Comment:
So, the flow is:

1. Try non-ssl to see if that fails
2. If non-ssl fails, try ssl to see if that fails

It was really a matter of trying to connection that generated the best (i.e., catchable) error first.  And, trying non-ssl against an ssl connection gave a better error than trying a ssl connection against a non-ssl connection.

I have no idea how to name this method, because (as you pointed out) we're really covering up a really bad situation.  We should really be asking the user whether the connection is SSL or not up front.  We sort of can't do that until we're supporting multiple end points per provider (or, manager, really).